index.ts

```diff
@@ -286,10 +286,27 @@ TEST: addTests('isQuickPR', [
 	'https://github.com/sindresorhus/refined-github/compare/test-branch?quick_pull=1',
 ]);
 
-export const isDraftPR = (): boolean => exists('#partial-discussion-header .octicon-git-pull-request-draft');
-export const isOpenPR = (): boolean => exists('#partial-discussion-header :is(.octicon-git-pull-request, .octicon-git-pull-request-draft)');
-export const isMergedPR = (): boolean => exists('#partial-discussion-header .octicon-git-merge');
-export const isClosedPR = (): boolean => exists('#partial-discussion-header :is(.octicon-git-pull-request-closed, .octicon-git-merge)');
+const prStateSelector = [
+	'.State',
+	'[class^="StateLabel"]',
+].join(',');
```

Comment:
```suggestion
];
```

If it doesn't work, just update `select-dom`

Reply:
<img width="906" alt="image" src="https://github.com/user-attachments/assets/a571a12e-378e-4fd1-ad19-12ca007d179a">

<img width="1306" alt="image" src="https://github.com/user-attachments/assets/b23c535a-4f99-4c51-b4f5-c51351120863">


<img width="1071" alt="image" src="https://github.com/user-attachments/assets/6d055f51-245a-43b9-a7ea-5cd43225615e">

Reply:
I'm referring to dropping `.join()`

Reply:
But it doesn't support string[].

<img width="630" alt="image" src="https://github.com/user-attachments/assets/7cb8597f-da29-49dc-80ad-ea538021fb19">

Reply:
Ah yeah because we're not using `select-dom`